src/core/lib/surface/channel.cc

```diff
@@ -150,7 +150,6 @@ absl::StatusOr<RefCountedPtr<Channel>> Channel::CreateWithBuilder(
     compression_options.enabled_algorithms_bitset =
         *enabled_algorithms_bitset | 1 /* always support no compression */;
   }
-
```

Comment:
Please restore this blank line, so that there's no change needed to this file in this PR.

Reply:
Done